src/main/kotlin/telegram/Handling.kt

```diff
@@ -111,45 +111,36 @@ object MafiaHandler {
     private suspend fun ContainerBlock.BasicContext.lobbyCommands() {
         connections.find { playerId == chatId }.singleOrNull()?.let { con ->
             simple(leaveGameCommand, menuCommand, startCommand, leaveGameLegacyCommand) {
-                val chat = ChatId.fromId(chatId)
-                bot.deleteMessage(chat, messageId ?: -1L)
-                val res = bot.sendMessage(
-                    chat,
+                bot.deleteMessage(ChatId.fromId(chatId), messageId ?: -1L)
+                val msgId = sendMessage(
+                    bot, chatId,
                     "Вы уверены, что хотите покинуть игру?"
                 )
-                if (res.isSuccess) {
-                    val msgId = res.get().messageId
-                    bot.editMessageReplyMarkup(
-                        chat,
-                        msgId,
-                        replyMarkup = inlineKeyboard {
-                            row {
-                                // todo replace -1L with messageId
-                                button(acceptLeaveCommand, -1L, msgId)
-                                button(deleteMsgCommand named "Нет", msgId)
-                            }
+                updateMessage(
+                    bot, chatId, msgId,
+                    inlineKeyboard {
+                        row {
+                            // todo replace -1L with messageId
+                            button(acceptLeaveCommand, -1L, msgId)
+                            button(deleteMsgCommand named "Нет", msgId)
                         }
-                    )
-                }
+                    }
+                )
             }
             any {
                 games.get(con.gameId)?.let { game ->
                     if (game.state == GameState.Game) {
-                        val chat = ChatId.fromId(game.hostId)
-                        val res = bot.sendMessage(
-                            chat,
+                        val msgId = sendMessage(
+                            bot, chatId,
```

Comment:
You either put all parameters n one line or each one on a separate line. Otherwise list of parameters becomes less readable.